expression/builtin_op.go

```diff
@@ -45,50 +45,37 @@ type andandFunctionClass struct {
 }
 
 func (c *andandFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
```

Comment:
the name `andand` is weird, can we change it to "logicalAnd" like "logicalXor" ?